bigbluebutton-html5/imports/ui/core/graphql/singletons/connectionStatus.ts

```diff
@@ -233,4 +244,8 @@ class ConnectionStatus {
   }
 }
 
-export default new ConnectionStatus();
+const connectionsStatusSingleton = new ConnectionStatus();
+
+window.teste = connectionsStatusSingleton;
```

Comment:
```suggestion
```

Let's revert this portion @Tainan404 -- seems you edited the file just to be able to test